### ISSUE TITLE
rest.rester( options ) still supprted?

### ISSUE THREAD

**Issue by reybango** [imrefazekas/connect-rest#37]
Hey team,

I was reading through Ethan Brown's Node.js book where he asks readers to use connect-rest to define an api. The issue is that in the code he references `rest.rester( options )` which generates the following error:

> app.use(rest.rester(apiOptions));
> ^
> TypeError: rest.rester is not a function

I looked for `rester` in the code for v2.9.0 but didn't see a reference to it. Is `rester()` a deprecated method? Asking because it's also mentioned in the docs here:

[https://github.com/imrefazekas/connect-rest#context](https://github.com/imrefazekas/connect-rest#context)
[https://github.com/imrefazekas/connect-rest#discovery-services](https://github.com/imrefazekas/connect-rest#discovery-services)
[https://github.com/imrefazekas/connect-rest#more-examples](https://github.com/imrefazekas/connect-rest#more-examples)

**Comment by billbuilder** [comment]
Dear @reybango 

That syntax has been removed from version 2.0 which was a great improvement in stability and performance and capabilities as well inducing changes in the API.
Please check the readme how can you use connect-rest:

```
var rest = Rest.create( options )

// adds connect-rest middleware to connect
connectApp.use( rest.processRequest() )

// defines a few sample rest services
rest.get('/books/:title/:chapter', functionN0 )

rest.post( { path: '/make', version: '>=1.0.0' }, functionN1 );

rest.post( [ '/act', '/do' ], functionN2 );
```

**Comment by imrefazekas** [comment]
@reybango 

"he asks readers to use connect-rest to define an api" ---that is very kind of him. 
connect-rest is purely a REST api focusing on business logic abstracting  the API away from connect middleware system. 

With connect-rest, you can express a lot of great things beyond the reach of express, however connect-rest do not care about the presentation layer at all, it is purely a REST api, nothing more.

**Comment by imrefazekas** [comment]
Feel free to drop in your business problem to be solved and we would be delighted to help :)

**Comment by reybango** [comment]
@billbuilder thank you for the info. I had a feeling that was the case since I didn't see the method anywhere. And I had already implemented the code you listed which didn't throw an error and worked. I just wanted to confirm is the `rester()` method was gone. :)

@imrefazekas Appreciate the reply. I've got the code to not throw errors now after reading through your docs so I'm made progress. And it looks like the references to the `rester()` were removed from the docs which is great! For context, here's the repo with Ethan's sample code from the book: [https://github.com/EthanRBrown/web-development-with-node-and-express/blob/master/ch15/meadowlark.js](https://github.com/EthanRBrown/web-development-with-node-and-express/blob/master/ch15/meadowlark.js). It's a popular book so I wanted to ensure to bring this up. :)

Now if I can make the REST endpoints work in conjunction with the vhost npm package, I'd be all set. :)
